### PR TITLE
Discord rich presence 

### PR DESCRIPTION
**Pull request by Lonsfor**
as noted by upstream launching the game **from** discord doesnt work but if you have the game open, status and chat game invites work normally. the only thing not tested is 'ask to join'.

for reference see https://github.com/flathub/com.discordapp.Discord/wiki/Rich-Precense-(discord-rpc)

### CODE REVIEW

**Comment by ghisvail** [comment]
Could you record a short video to see how that looks like. I must admit I
have no idea what this feature is.

File: org.srb2.SRB2Kart.yaml [hunk 1]
```diff
@@ -11,20 +11,47 @@ finish-args:
   - --socket=wayland
   - --device=all
   - --persist=.srb2kart
+  - --filesystem=xdg-run/app/com.discordapp.Discord:create
+cleanup:
+  - /include
+  - /lib/pkgconfig
+  - /lib/cmake
+  - /man
+  - /share/doc
```
Comment (ghisvail) [inline org.srb2.SRB2Kart.yaml:20]: It is better practice to have individual modules do their own cleanup.

**Review by ghisvail — CHANGES_REQUESTED** [review]

File: org.srb2.SRB2Kart.yaml (hunk 1 above)
Comment (Lonsfor) [reply org.srb2.SRB2Kart.yaml]: that should be better

**Comment by cbm755** [comment]
Sorry, I have no opinion on this.  Nor any understanding of how to test it.

But I'm not opposed to merging: @Lonsfor certainly seems to know what's going on here.

**Comment by ghisvail** [comment]
Thanks @Lonsfor

**Pull request merged**